lib/pacemaker/pcmk_fence.c

```diff
@@ -157,8 +161,10 @@ pcmk__fence_action(stonith_t *st, const char *target, const char *action,
 #ifdef BUILD_PUBLIC_LIBPACEMAKER
 int
 pcmk_fence_action(stonith_t *st, const char *target, const char *action,
-                  const char *name, unsigned int timeout, unsigned int tolerance) {
-    return pcmk__fence_action(st, target, action, name, timeout, tolerance);
+                  const char *name, unsigned int timeout, unsigned int tolerance,
+                  int delay)
+{
+    return pcmk__fence_action(st, target, action, name, timeout, tolerancei, delay);
```

Comment:
extra "i". doesn't matter now but will save us a moment of confusion later :)

Reply:
Wow, good catch! You have eagle eyes :-)